docs/configuration/README.md

```diff
@@ -7,272 +7,4 @@ See [the `Dockerfile` in the `signals-amsterdam` repository](https://github.com/
-
-### INCIDENT_PRIVATE_ENDPOINT
-
-URL for the private incident endpoint. Used to `POST` incident data for authorized users.<br>
-See https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/post_signals_v1_private_signals_ for reference.
-
-### INCIDENT_PUBLIC_ENDPOINT
-
-URL for the public incident endpoint. Used to `POST` incident data for unauthorized users.<br>
-See https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/post_signals_v1_public_signals for reference.
-
-### OIDC_AUTH_ENDPOINT
-
-URL for the oAuth2 service.
-
-### OIDC_CLIENT_ID
-
-oAuth2 `client_id` service parameter value.
-
-### OIDC_RESPONSE_TYPE
-
-oAuth2 `response_type` service parameter value.
-
-### OIDC_SCOPE
-
-oAuth2 `scope` service parameter value (specific to `signals-amsterdam` and `signals-weesp` implementations).
-
-### OVL_KLOKKEN_LAYER
-
-GeoJSON URL for the retrieval of public clock location data. Used by the incident form for showing clock locations on a map.<br>
-No Swagger documentation available.
-
-### OVL_VERLICHTING_LAYER
-
-GeoJSON URL for the retrieval of street light data. Used by the incident form for showing street light locations on a map.<br>
-No Swagger documentation available.
-
-### PERMISSIONS_ENDPOINT
-
-URL for the private permissions endpoint. Used by the settings module to `GET` the full set of permissions.<br>
-No Swagger documentation available.
-
-### PREDICTION_ENDPOINT
-
-URL for the public machine learning prediction endpoint. Used by the incident form to get an indication of (sub)category based on user input.<br>
-No Swagger documentation available.
-
-### QUESTIONS_ENDPOINT
-
-URL for the public question endpoint. Used to `GET` question configuration based on the value of the [fetchQuestionsFromBackend](#fetchQuestionsFromBackend) settings prop.
-
-### ROLES_ENDPOINT
-
-URL for the private roles endpoint. Used by the settings module to `GET`, `PATCH` and `POST` roles.<br>
-No Swagger documentation available.
-
-### SEARCH_ENDPOINT
-
-URL for the private search endpoint. Used by the incident management module to retrieve incident by search query.<br>
-No Swagger documentation available.
-
-### TERMS_ENDPOINT
-
-URL for the private terms endpoint. Used by the incident management module to `GET` and `POST` default response texts for incident handling.<br>
-See https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/get_signals_v1_public_terms_categories for reference.
-
-### USERS_ENDPOINT
-
-URL for the private users endpoint. Used by the settings module to `GET`, `PATCH` and `POST` users.<br>
-See https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/get_signals_v1_private_users_ for reference.
+Properties and their descriptions are defined in [mounted into the container](../../internals/schemas/environment.conf.schema.json).
```

Comment:
Might as well remove the entire file and replace it with `docs/schemas/README.md`?

Reply:
Sounds good to me! :+1: